lib/shared/types.js

```diff
@@ -189,11 +189,44 @@ module.exports = {};
  * @property {number} warningCount Number of warnings for the result.
  * @property {number} fixableErrorCount Number of fixable errors for the result.
  * @property {number} fixableWarningCount Number of fixable warnings for the result.
+ * @property {Stats} Stats The performance statistics collected with the `stats` flag.
```

Comment:
```suggestion
 * @property {Stats} [stats] The performance statistics collected with the `stats` flag.
```

Lowercase `stats` is the property name. `[]` because it's optional (it may or may not be present depending on whether the option was used).